src/index.d.ts

```diff
@@ -4000,6 +4000,120 @@ declare namespace admin.messaging {
+     * is set to `true`, the default value is used instead of the user-specified `vibrate_timings`.
+     */
+    vibrateTimingsMillis?: number[];
+
+    /**
+     * If set to `true`, use the Android framework's default vibrate pattern for the 
+     * notification. Default values are specified in [`config.xml`](https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/values/config.xml). 
+     * If `default_vibrate_timings` is set to `true` and `vibrate_timings` is also set, 
+     * the default value is used instead of the user-specified `vibrate_timings`.
+     */
+    defaultVibrateTimings?: boolean;
+
+    /**
+     * If set to `true`, use the Android framework's default sound for the notification. 
+     * Default values are specified in [`config.xml`](https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/values/config.xml).
+     */
+    defaultSound?: boolean;
+
+    /**
+     * Settings to control the notification's LED blinking rate and color if LED is 
+     * available on the device. The total blinking time is controlled by the OS.
+     */
+    lightSettings?: LightSettings;
+
+    /**
+     * If set to `true`, use the Android framework's default LED light settings 
+     * for the notification. Default values are specified in [`config.xml`](https://android.googlesource.com/platform/frameworks/base/+/master/core/res/res/values/config.xml). 
+     * If `default_light_settings` is set to `true` and `light_settings` is also set, 
+     * the user-specified `light_settings` is used instead of the default value.
+     */
+    defaultLightSettings?: boolean;
+
+    /**
+     * Set the visibility of the notification. Must be either `private`, `public`, 
+     * or `secret`. If unspecified, default to `private`.
+     */
+    visibility?: ('private' | 'public' | 'secret');
+
+    /**
+     * Sets the number of items this notification represents. May be displayed as a 
+     * badge count for Launchers that support badging. See [`NotificationBadge`(https://developer.android.com/training/notify-user/badges). 
+     * For example, this might be useful if you're using just one notification to
+     * represent multiple new messages but you want the count here to represent
+     * the number of total new messages. If zero or unspecified, systems 
+     * that support badging use the default, which is to increment a number 
+     * displayed on the long-press menu each time a new notification arrives.
+     */
+    notificationCount?: number;
+  }
+
+  /**
+   * Represents settings to control notification LED that can be included in
+   * {@link admin.messaging.AndroidNotification}.
+   */
+  interface LightSettings {
+    /**
+     * Required. Set color of the LED in `#rrggbb` or `#rrggbbaa` format.
+     */
+    color: string;
+
+    /**
+     * Required. Along with `light_off_duration`, define the blink rate of LED flashes.
+     */
```

Comment:
@egilmorez : Updated the docs to add `Required` here. Also, should the verbs on docs be singular or plural?
Ex: `Set` vs `Sets` or `define` vs `defines`

Reply:
I think the verbs should all be singular unless there's a really obvious case for plural.  Since most comments describe a single property or method rather than groups of them, we can consider singular the "default" . :)